pkg/karmadactl/describe.go

```diff
@@ -2,263 +2,115 @@ package karmadactl
 
 import (
 	"fmt"
-	"strings"
 
 	"github.com/spf13/cobra"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/api/meta"
-	utilerrors "k8s.io/apimachinery/pkg/util/errors"
-	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/cli-runtime/pkg/resource"
+	kubectldescribe "k8s.io/kubectl/pkg/cmd/describe"
 	cmdutil "k8s.io/kubectl/pkg/cmd/util"
 	"k8s.io/kubectl/pkg/describe"
-	"k8s.io/kubectl/pkg/util/templates"
 
 	"github.com/karmada-io/karmada/pkg/karmadactl/options"
 )
 
-var (
-	describeExample = templates.Examples(`
```

Comment:
Thanks for your contribution  @wlp1153468871. Only a small sugguestion, others are nice.

keeping this part of the code, and use it like the following,

```go
Example:               fmt.Sprintf(describeExample, parentCommand),
```

Reply:
Done